e2e/test/PoolingOverAmqp.cs

```diff
@@ -65,90 +92,123 @@ public static class PoolingOverAmqp
                     TestDevice testDevice = await TestDevice.GetTestDeviceAsync($"{devicePrefix}_{i}_").ConfigureAwait(false);
                     DeviceClient deviceClient = testDevice.CreateDeviceClient(transportSettings, authScope);
 
-                    var amqpConnectionStatusChange = new AmqpConnectionStatusChange();
+                    var amqpConnectionStatusChange = new AmqpConnectionStatusChange(devicePrefix);
                     deviceClient.SetConnectionStatusChangesHandler(amqpConnectionStatusChange.ConnectionStatusChangesHandler);
 
                     testDevices.Add(testDevice);
                     deviceClients.Add(deviceClient);
                     amqpConnectionStatuses.Add(amqpConnectionStatusChange);
 
-                    operations.Add(initOperation(deviceClient, testDevice));
+                    operations.Add(deviceClient.OpenAsync());
```

Comment:
Only opening the client will not cause all links to be opened. It might be a better idea to add this check inside the `CombinedClientOperationsPoolAmqpTests`; or alleast have them perform all operations

Reply:
Sure, I can refactor this test so that CombinedClientOperationsPoolAmqpTests behaves like it used to (cares about connection status), while the other amqp pool tests don't